src/microbe_stage/membrane_system.h

```diff
@@ -14,6 +14,9 @@
 
 namespace thrive {
 
+//enumerable for membrane type
+enum class MEMBRANE_TYPE { membrane, wall, chitin };
```

Comment:
These enum values should be UPPERCASE to match the style. (I'm not sure if this is in the style guide).